Dockerfile

```diff
@@ -16,6 +16,10 @@ RUN apk --update add \
     less \
     libsodium
 
+COPY entrypoint.sh /usr/bin/
+RUN chmod +x /usr/bin/entrypoint.sh
```

Comment:
What’s the reason to apply the execute permission only after copying to the container? Why can’t we just check in the script with the correct permissions in the beginning?

Reply:
Because it shouldn't be executed in a non-virtualized environment like your local machine.